Scripts/Runtime/Entities/Tasks/Jobs/JobConfig/UpdateJobConfig.cs

```diff
@@ -0,0 +1,50 @@
+using Anvil.Unity.DOTS.Jobs;
+
+namespace Anvil.Unity.DOTS.Entities.Tasks
+{
+    internal class UpdateJobConfig<TInstance> : AbstractResolvableJobConfig
```

Comment:
I would say no for now. 

I did a quick check and I only found the `JobConfigFactory` has a `///` comment for the overall class to explain it. Otherwise there are only inheritdoc refs for the public facing interfaces of JobConfigs.